third-party/genuine-fantasy-press/forgotten-secrets/invocations.xml

```diff
@@ -1262,6 +1262,7 @@
 		<prerequisite>Wild Huntsman patron</prerequisite>
 		<requirements>ID_GFP_COFSA_ARCHETYPE_OTHERWORLDLY_PATRON_WILD_HUNTSMAN</requirements>
 		<description>
+			<p class="flavor">Prerequisite: Wild Huntsman patron</p>
```

Comment:
you can use the `<p><em>` here instead of the flavor class. If I ever change the flavor class this should not change with it.

Reply:
Is there a quick way to change all the references to `<p><em>`?